tuxemon/event/conditions/check_world.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2024 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import final
+
+from tuxemon.event import MapCondition, get_npc
+from tuxemon.event.eventcondition import EventCondition
+from tuxemon.graphics import string_to_colorlike
+from tuxemon.session import Session
+from tuxemon.states.world.worldstate import WorldState
+
+logger = logging.getLogger(__name__)
+
+
+@final
+@dataclass
+class CheckWorldCondition(EventCondition):
+    """
+    Check some world's parameter against a given value.
+
```

Comment:
it took me a minute to understand what "layer" and "bubble" are used for.  i think we should include some additional information about the parameters.  maybe something like this:

```
layer: color value which is used to overlay the world
bubble: speech bubble of an npc
```
did i get that right?

Reply:
you're right, I'm going to add it

Reply:
Added below, just above the example